### PR TITLE
"Document: drag event":  example text selection for Firefox users

### LINKED ISSUE #2941: Issue with "Document: drag event": example behaves strange around text selection
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Document/drag_event

#### What information was incorrect, unhelpful, or incomplete?

The example behaves incorrectly in certain circumstances. This isn't exactly a critical bug, but people may inadvertently copy the example without realising it fails on edge-cases. Basically, it's possible to select the text inside the dive and drag that. I suppose in the `dragstart` event handler there should be a verification that the dragged element is the intended div and nothing else?

In firefox the text selection is clearly visible, but in Chrome there isn't such feedback.

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/API/Document/drag_event#examples

#### What did you expect to see?

Either the text is draggable, but not accepted by the dropzone, and/or the text is not selectable

#### Did you test this? If so, how?

firefox:
reviews

chrome:
chrome

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/document/drag_event`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Document/drag_event
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/document/drag_event/index.html
* Last commit: https://github.com/mdn/content/commit/9917bf17a97973166bb3cb3812a496e4117a778f
* Document last modified: 2021-01-11T13:30:57.000Z

</details>

### PR DESCRIPTION
**Pull request by sidverma32**
#### Summary

"Document: drag event":  example  section Text was getting selected and getting dragged with the <div> for Firefox users
#### Motivation

Added a specific check for `MozUsers` to prevent the Text Selection while performing drag event.
#### Related issues

Fixes [2941](https://github.com/mdn/content/issues/2941)
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks for this PR, @sidverma32 !

I could only manage to select text (and therefore reproduce this) by clicking the text and pressing Ctrl-A, is there another way? Anyway, this PR does seem to make that impossible, so 👍 .

But could we use `user-select` instead of the `-moz` prefixed version, and can we do it in the CSS rather than setting it in JS:

```css
body {
  user-select: none;
}
```

?

**Comment by sidverma32** [comment]
> Thanks for this PR, @sidverma32 !
> 
> I could only manage to select text (and therefore reproduce this) by clicking the text and pressing Ctrl-A, is there another way? Anyway, this PR does seem to make that impossible, so 👍 .
> 
Sorry i couldn’t understand this "Impossible" which thing??
> But could we use `user-select` instead of the `-moz` prefixed version, and can we do it in the CSS rather than setting it in JS:
> 
> ```css
> body {
>   user-select: none;
> }
> ```
> 
> ?

Yes, we can do it via CSS also.

**Comment by wbamberg** [comment]
> Sorry i couldn’t understand this "Impossible" which thing??

Sorry to be unclear. I mean, with the change in your PR it is impossible to select the text any more.

**Comment by sidverma32** [comment]
> 

Yeah and i guess that was the expectation also!! 

So, the final call is to handle this using **CSS** @wbamberg ? I will update the PR accordingly

**Comment by wbamberg** [comment]
> > Sorry to be unclear. I mean, with the change in your PR it is impossible to select the text any more.
> 
> Yeah and i guess that was the expectation also!!
> 
> So, the final call is to handle this using **CSS** @wbamberg ? I will update the PR accordingly

Yes please! :)

File: files/en-us/web/api/document/drag_event/index.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ See this code in a [JSFiddle demo](https://jsfiddle.net/zfnj5rv4/) or interact w
 ### CSS
 
 ```css
+body {
+  user-select: none; /*Prevent the text from getting selectable for Firefox users*/
```
Comment (wbamberg) [inline files/en-us/web/api/document/drag_event/index.md:68]:
```suggestion
body {
  /* Prevent the user selecting text in the example */
  user-select: none;
```

**Review by wbamberg — APPROVED** [review]
👍 thank you for the fix @sidverma32 !

**Pull request merged**